### DISCUSSION TITLE
Replace light/dark toggle with icons

### DISCUSSION THREAD

**Discussion by Dannyholt** [facebook/docusaurus#6714] [Ideas]
Can you guys consider replacing the current light/dark mode toggle with separate icons similar to what Ionic have done on their docs.

https://ionicframework.com/docs

I can't speak for anyone else, but for me, the toggle just doesn't go with the rest of the style of the nav and takes too much space when compared with a single icon.

**Comment by Josh-Cena** [comment]
I have the same idea. Sent #6717 to discuss this change along with a few others.

**Reply by slorber** [reply]
also agree with this 👍 , and having a smaller theme icon would also save some useful navbar space
